.github/workflows/release_draft.yml

```diff
@@ -75,35 +78,35 @@ jobs:
 
           tag-name: ${{ github.event.inputs.tag-name }}
 
-  release:
-    needs: tag
+  generate-release-notes:
+    needs: check-tag
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v4.1.1
         with:
           fetch-depth: 0
-          ref: refs/tags/${{ github.event.inputs.tag-name }}
 
       - uses: actions/setup-python@v5.1.1
         with:
           python-version: '3.11'
 
-      - name: Generate release notes
-        id: release_notes_generator
+      - name: Generate Release Notes
+        id: generate_release_notes
         uses: AbsaOSS/generate-release-notes@master
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           tag-name: ${{ github.event.inputs.tag-name }}
-          chapters: |
-            [
-              {"title": "Breaking Changes 💥", "label": "breaking-change"},
-              {"title": "New Features 🎉", "label": "feature"},
-              {"title": "New Features 🎉", "label": "enhancement"},
-              {"title": "Bugfixes 🛠", "label": "bug"}
-            ]
+          chapters: '[
```

Comment:
Why is the array a string?

Reply:
Will be fixed in Issue #https://github.com/AbsaOSS/generate-release-notes/issues/85.